Gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+ruby '~> 3.3.0'
+
+gem 'fluentd', '~> 1.16.1'
```

Comment:
pin exact `fluentd` version to match `LABEL konflux.additional-tags`

```ruby
gem 'fluentd', '1.16.1'
```

and to `bundle update` to see `Gemfile.lock` updated, as it's `1.16.9` now.

Reply:
Good catch, thank you!